SampleGeneration/20251020_ForestReducer_2025OpenCharmUPC/ReduceForest_Dzero.cpp

```diff
@@ -115,7 +115,8 @@ int main(int argc, char *argv[]) {
     TFile InputFile(InputFileName.c_str());
 
     HiEventTreeMessenger MEvent(InputFile); // hiEvtAnalyzer/HiTree
-    PbPbUPCTrackTreeMessenger MTrackPbPbUPC(InputFile); // ppTracks/trackTree
+    PbPbUPCTrackTreeMessenger MTrackPbPbUPC(InputFile, (Year == 2025)? "PbPbTracks/trackTree":
+                                                                       "ppTracks/trackTree"); // ppTracks/trackTree
```

Comment:
As they won't appear at the same time, I would suggest not using `Year` as the toggle, while instead, do "use `PbPbTracks/trackTree` if it exists, else if `ppTracks/trackTree` exists, use it" to automatically use the correct tree. 
Here is an example to check if a tree exists
https://github.com/boundino/csetup/blob/master/xjjlibs/xjjroot/xjjanauti.h#L508-L521